lib/pbench/cli/server/options.py

```diff
@@ -0,0 +1,31 @@
+import click
+
+from pbench.cli.server import CliContext
+
+
+def common_options(f):
+    f = _pbench_server_config(f)
+    return f
```

Comment:
This is kind of a triviality, but this code is poor style/practice:  there is no good reason for the input of the function to be overwritten with the output.  (We're not running on a 80386 CPU with such limited resources that we have to help the compiler reuse processor registers!)

However, more significantly, there is no clue here as to what the input to the function is supposed to be, nor what the output is supposed to be, nor what the overall purpose of this function is.  Since this is the external interface to this module, it would be appropriate to have a docstring.

And, since this is new code, it would be good to use Python's type hints for the input and return value (ditto for `_pbench_server_config()`, as well).

Most people aren't familiar with the details of how Python's function decorators work, so it might be worth being a little bit verbose when pointing out that this function receives a function as an argument and returns another function (which will call the input) as its output.